FSJ_django20_project/FSJ/models_application.py

```diff
@@ -0,0 +1,21 @@
+import uuid
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from .models_award import Award
+from .models_FSJUser import FSJUser
+
+class Application(models.Model):
+    application_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    award = models.ForeignKey(Award, on_delete=models.CASCADE, related_name='applications')
+    student = models.ForeignKey(FSJUser, on_delete=models.CASCADE, related_name='applications', to_field='ccid')
+    is_submitted = models.BooleanField(verbose_name = _("Is Submitted"))
+    application_file = models.FileField(null=True, blank=True, upload_to='documents/')
+    
+    def __str__(self):
+        return self.student.ccid + "'s application for " + self.award.award_name    
```

Comment:
No way to translate this English text.

Reply:
Str function is for debugging purposes only and should not show up anywhere in application